integration/offline_test.go

```diff
@@ -0,0 +1,91 @@
+package integration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testOffline(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack()
+		docker = occam.NewDocker()
+	})
+
+	context("when building a simple app offline", func() {
+		var (
+			image     occam.Image
+			container occam.Container
+
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("creates a working OCI image", func() {
+			var err error
+			source, err = occam.Source(filepath.Join("testdata", "offline_app"))
+			Expect(err).NotTo(HaveOccurred())
+
+			image, _, err = pack.WithVerbose().Build.
+				WithBuildpacks(
+					settings.Buildpacks.MRI.Offline,
+					settings.Buildpacks.Bundler.Offline,
+					settings.Buildpacks.BundleInstall.Offline,
+					settings.Buildpacks.BuildPlan.Online,
+				).
+				WithNetwork("none").
+				WithNoPull().
+				Execute(name, source)
+			Expect(err).NotTo(HaveOccurred())
+
+			container, err = docker.Container.Run.
+				WithCommand("env && echo \"bundle -> $(which bundle)\" && cat $(which bundle) && bundle exec rackup -o 0.0.0.0").
```

Comment:
Since you aren't asserting on any of the output from this command, I think this can be simplified a bit.
```suggestion
				WithCommand("bundle exec rackup -o 0.0.0.0").
```